duo-universal-sdk/pom.xml

```diff
@@ -115,12 +115,13 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
-            <version>1.18.10</version>
+            <version>1.18.16</version>
+            <scope>compile</scope>
```

Comment:
Thank you for pointing out this potential optimization! I _believe_ we actually want the `provided` scope here, as my understanding is that `compile` is the default and wouldn't actually change the behavior. https://stackoverflow.com/questions/29385921/maven-scope-for-lombok-compile-vs-provided

Reply:
Absolutely; updated.

By the way, I didn't go that far but you should also take a look at the project hierarchy and the parent tags for the sub-modules. Maven's quite forgiving, but I am not so sure if the current structure is entirely accurate. Parent references seem to be missing or are misplaced.